listenbrainz/webserver/static/js/src/recommendations/RecommendationCard.tsx

```diff
@@ -177,45 +219,47 @@ export default class RecommendationCard extends React.Component<
         <div className="col-xs-1 text-center">
           <div className="recommendation-controls">
             <>
-              <FontAwesomeIcon
-                icon={faEllipsisV as IconProp}
-                className="dropdown-toggle"
+              <button
+                className="btn"
                 id="recommendationControlsDropdown"
                 data-toggle="dropdown"
                 aria-haspopup="true"
                 aria-expanded="true"
-              />
+                type="button"
+              >
+                <FontAwesomeIcon icon={icon as IconProp} /> {text}
```

Comment:
@MonkeyDo I haven't used Font Awesome regular icons in this commit. Once we are okay with the general flow of it then we can move to the styling part. Can you please try and run this on your machine and play with the feedback buttons and let me know if we are on the same page regarding the general flow?

Reply:
Posting a video on IRC for your reference!

Reply:
I think the general flow works well. It's easy enough for users to understand without any room for doubt.